doc/developer/interface/custom-login/index.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: default
+---
+
+
+Custom Login Page
+==========================================================
+
+The default landing / login page for IRIDA can be replaced by an institution without modifying the existing one.
+
+Create a html file called `login.html` on your files system.  The default path IRIDA will check is `/etc/irida/templates`, but can be configured in the configuration file `src/main/resources/configuration.properties` by updating the `ui.templates` property,
+
+```properties
+# Overwritten UI templates
+ui.templates=/etc/irida/templates/
+```
+
+### Use IRIDA base login form.
+
+IRIDA uses a React component as its login form and can be easily rendered on and DOM node.  The basic layout can be found in the code snippet below:
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" th:href="@{/dist/css/login.bundle.css}" />
+    <script th:inline="javascript">
+      window.PAGE = {
+        BASE_URL: /*[[@{"/"}]]*/ "/"
+      };
+    </script>
+  </head>
+  <body>
+    <div id="login-root"></div>
+    <script th:src="@{/dist/js/login.bundle.js}"></script>
+  </body>
+</html>
+```
+
+ * CSS and JS links are required.  This will include all `React` and styles required.  the `th:` is for our server templating language Thymeleaf to set the url's properly.
+ * `BASE_URL` is required by the script file in order to properly set the servlet context for the current installation.
+ * The div with id `login-root` should be placed on the dom element that you want React to render the login form.
+
+#### Example of Custom login page
+
+![Custom Login Page](images/irida-custom-login.png)
+
+### Create a custom login form
+
+Login form basic skeleton:
```

Comment:
I'm not really sure what this is for.  Is it for if they want to completely re-write the login form rather than use the `<div id="login-root"></div>` section?  If so that needs more detail please.

Reply:
Updated in 169dd1396a046e4bfd0079d5c6672be5088d722f